parsing/dialects/intermediate/dialects/dialect1/src/test/java/org/apache/calcite/test/Dialect1ValidatorTest.java

```diff
@@ -28,4 +38,15 @@
         SqlTestFactory.INSTANCE
             .with("parserFactory", Dialect1ParserImpl.FACTORY));
   }
+
+  @Test public void testCaretNegation() throws SqlParseException {
```

Comment:
We probably don't want to do all of this setup for each test case. I've moved some of this into a common location in: https://github.com/googleinterns/calcite/pull/311

Suggest waiting for that to be merged and then remove a lot of this setup logic.